src/components/site/SummaryList.md

```diff
@@ -41,40 +41,48 @@ const publicWorksSummaryItems = [
   }
 ];
 
-<SummaryList items={publicWorksSummaryItems} />;
+<SummaryList id="summaryList-PublicWorks" items={publicWorksSummaryItems} />;
```

Comment:
we should probably remove this from the example, especially since the snippet does not have an id.

Reply:
removed